youtube_dl/extractor/xvideos.py

```diff
@@ -76,13 +76,24 @@ class XVideosIE(InfoExtractor):
     }, {
         'url': 'https://de.xvideos.com/video4588838/biker_takes_his_girl',
         'only_matching': True
+    }, {
+        'url': 'https://www.xvideos2.com/video4588838/biker_takes_his_girl',
+        'only_matching': True
+    }, {
+        'url': 'https://www.xvideos3.com/video4588838/biker_takes_his_girl',
+        'only_matching': True
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage = self._download_webpage(
-            'https://www.xvideos.com/video%s/' % video_id, video_id)
+        HOSTS = ('xvideos', 'xvideos2', 'xvideos3')
+        self.report_download_webpage(video_id)
```

Comment:
Remove.

Reply:
I dont understand. Do you want me to remove the self.report_download_webpage, but then every time it checks the host it will out the "downloading webpage" line, this way it only does it once.

Reply:
Again: remove. Any download attempt must have clear explicit message.